### ISSUE TITLE
"What links here" display

### ISSUE THREAD

**Issue by enterprisey** [WPAFC/afch-rewrite#123]
In the "Accepting..." user interface, we should show a list of articles that link to the future title of the article, so the reviewer can check them.

Original description:
> During acceptance, check for incoming redirects to the target from other articles. If there are, give reviewers a chance to cancel or put the page "under review" so the incoming links can be checked for relevance.

From [davidwr, 15:47, 4 December 2020 (UTC)](https://en.wikipedia.org/wiki/Wikipedia_talk:WikiProject_Articles_for_creation/2020_5#2_AFC_script_feature_suggestions) ([permalink](https://en.wikipedia.org/w/index.php?diff=992299958))

**Comment by primefac** [comment]
I don't see this as a particularly high priority. Why do we need to check the articles that link to a potential article?

**Comment by enterprisey** [comment]
Yes, this is very low-priority. Perhaps just something the reviewer may be interested in, in the same way I'd check incoming links when creating a new article.
